surveys/sphinx-survey-2025.yml

```diff
@@ -192,8 +222,8 @@ questions:
   - "Stability"
   - "Variety of features"
 
-# Motivation: Reflect on priorities for documentation in general. 
-- title: "Why did you choose the engine you use for your documentation?"
+# Motivation: Reflect on priorities for documentation in general.
+- title: "Why did you choose the tool/engine you use for your documentation?"
```

Comment:
Is "tool" only is sufficient?

Reply:
Now that I see this question, I think the "What documentation tools/engines do you use?" question needs to be refactored.

It seems like the valuable info would be determining whether respondents primarily use Sphinx or something else, and why

In the "What documentation tools/engines do you use?" it doesn't make sense to list Sphinx and Rustdoc (for example) because we use them both and we use them for different reasons

So I think the question as it currently stands is going to generate a lot of misleading noise

Reply:
Well, OK, now I see that there are questions below that are focused on Sphinx. 

I'll leave my previous comment as-is: I suggest rethinking this question and the "What documentation tools/engines do you use?" one

Reply:
I've changed my mind on this question. I think it's fine to use a single question in order to capture clusters of tools that are commonly used together, as Tania explained